admin_manual/configuration_server/logging_configuration.rst

```diff
@@ -131,34 +135,55 @@ Log field breakdown
 
 Empty value are written as two dashes: ``--``.
 
-Admin audit log
----------------
+Admin audit log (Optional)
+--------------------------
+
+By enabling the **admin_audit** app, additional information about various events can be logged. Similar to the normal logging, the audit log can be provided to any of the existing logging mechanisms in :file:`config/config.php`. The default behavior, if no parameters are specified after the app is enabled, is ``file`` based logging to a file called ``audit.log`` stored in the ``datadirectory``.
 
-If ``loglevel`` in ``config.php`` is set to ``2`` or higher, audit logging needs to be triggered explicitly by adding the following setting to to ``config.php``:
+If you wish to override this and log to syslog instead the following would be one approach:
 
 ::
 
-	'log.condition' => [
-		'apps' => ['admin_audit'],
+	"log_type_audit" => "syslog",
+	"syslog_tag_audit" => "Nextcloud",
+	"logfile_audit" => "",
+
+Log level interaction
+~~~~~~~~~~~~~~~~~~~~~
+
+If system ``loglevel`` in ``config.php`` is set to ``2`` or higher, audit logging needs to be triggered explicitly by adding the following setting to to ``config.php``:
+
+::
+
+	"log.condition" => [
+		"apps" => ["admin_audit"],
 	],
 
+Find detailed documentation on auditable events for enterprises in our `customer portal <https://portal.nextcloud.com/article/using-the-audit-log-44.html>`_.
+
+Integrating into the Web Interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The built-in NC ``logreader`` app (which is what provides the *Administration settings->Logging* interface) only accesses the file-based ``nextcloud.log``. The **admin_audit** app log output, however, can be integrated into the web interface by configuring it to *also* log to the ``nextcloud.log``.
```

Comment:
Ah I see. Better would be something like https://github.com/nextcloud/logreader/pull/857 though

Reply:
That PR looks quite useful if one doesn't wish to combine their on-disk audit log with the normal Nextcloud log.

The confusing thing has been that screenshot from the logreader app repository README - (it's referenced in nextcloud/logreader#129 and occasionally on the forum). It shows everything merged together. :-)

So it turns out it's totally been possible all along.